website/blog/2024-01-23-Code-execution-in-docker/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Code execution inside docker container"
+authors:
+  - olgavrou
+tags: [AutoGen]
+---
+
+
+## TLDR
+
+AutoGen 0.2.8 addresses security vulnerabilities, focusing on informing users about its operations and empowering them to make informed decisions regarding code execution.
```

Comment:
I'm not sure I would use "security vulnerabilities" here. It's more of about making a recommended operating practice a default one.

Reply:
maybe:

AutoGen 0.2.8 enhances operational safety by making 'code execution inside a Docker container' the default setting.